src/libtrx/include/libtrx/game/music/const.h

```diff
@@ -0,0 +1,3 @@
+#pragma once
+
+#define MAX_CD_TRACKS 64
```

Comment:
Let's name this `MAX_MUSIC_TRACKS`.

Reply:
Thanks, done. Added the commit and changelog entries for #2460 as well.